test/run/integration-tests-from-docker-container.sh

```diff
@@ -13,4 +13,5 @@ run_pants_python_tests() {
 
 # -rA means "report stdout/stderr for all tests, not just failing tests"
```

Comment:
(yes, `-rA` is the longest form available)

Reply:
Pre-empting my usual code review comments, I see :joy: